docs/src/operate/configuration/config-mapper-mqtt-topics.md

```diff
@@ -0,0 +1,64 @@
+# How to configure MQTT topics to subscribe for mappers
+
+The cloud-specific mappers subscribe to the reserved MQTT topics and convert incoming MQTT messages to cloud-specific messages.
+In an advanced use case, such as using more than one cloud mappers for the same device,
+you may want to customize the MQTT topics that each cloud mapper subscribes to.
+
+The `tedge config` command and the keys `c8y.topics`, `az.topics`, and `aws.topics` are usable for this use-case.
+
+```admonish note
+This guide uses `c8y.topics`, `TEDGE_C8Y_TOPICS`, and `tedge-mapper-c8y.service` as an example.
+For other cloud mappers, use the keys in the table.
+```
+
+| Cloud          | tedge config key | Environmental variable | systemctl service        |
+|----------------|------------------|------------------------|--------------------------|
+| Cumulocity IoT | c8y.topics       | TEDGE_C8Y_TOPICS       | tedge-mapper-c8y.service |
+| Azure IoT      | az.topics        | TEDGE_AZ_TOPICS        | tedge-mapper-az.service  |
+| AWS IoT        | aws.topics       | TEDGE_AWS_TOPICS       | tedge-mapper-aws.service |
+
+## Check the subscribed MQTT topics
+
+First, check which MQTT topics are subscribed by a cloud mapper. Run:
+
+```shell
+tedge config get c8y.topics
+```
+
```

Comment:
An example response might be useful for the user to know what to expect.

Reply:
Added